src/common_cli/version.clj

```diff
@@ -10,11 +11,7 @@
 
 (defn jar-file-stream
   [path-in-jar]
-  (.openStream
-   (java.net.URL.
-    (str "jar:"
-         (jar-domain)
-         path-in-jar))))
```

Comment:
Does this work the same? The old one appears to return an open stream, where the latter looks to only return the URI object. I'm not even sure where we use this function, but nonetheless thought I should ask.

Reply:
Ah, good catch. I think I got a little careless there. I'll check into it and fix it.

Reply:
I think it should be fixed now.